src/lib/migration/chainCache.ts

```diff
@@ -0,0 +1,50 @@
+import { ServiceFactory, LocalStorageService } from "../classes";
+import { ChainData } from "../classes/services/chainsService";
+
+const CURRENT_PERSISTED_DATA_VERSION = 2;
+/**
+ *
+ */
+export function checkAndMigrate(): void {
+    const storageService = ServiceFactory.get<LocalStorageService>(LocalStorageService.ServiceName);
+    const version = storageService.load("persisted data version") ?? 1; // Get the current version
```

Comment:
we have too many conventions in the local storage key name, lets unify them all 🙏🏼 
![image](https://github.com/iotaledger/wasp-dashboard/assets/3624944/318bb61c-0bc4-44a3-884d-e1fdf583ed41)
 I think `this-that` works fine

Reply:
Also we should avoid magic strinngs eg `persisted data version`